sorting.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+import random
+
+
+def insertion_sort(list):
```

Comment:
This is a huge no no in programming. Never name a variable a built in keyword name.